### ISSUE TITLE
Incorrect Kanon sprite

### ISSUE THREAD

**Issue by Knurek-NC** [07th-mod/umineko-question#36]
![20161009221721_1](https://cloud.githubusercontent.com/assets/22718168/19223389/a4233c58-8e6e-11e6-8dad-bead75c59b6f.jpg)

Looking at the sprite pictures, quite a lot of them are using the Pachinko data:
all are in big\bmp\tati\kan\1\

ka2_defa2b.png
ka2_fumana1b.png
ka2_fumana2b.png
ka2_ikaria1.png
ka2_ikaria1b.png
ka2_ikaria2.png
ka2_ikaria2b.png
ka2_ikaria3.png
ka2_ikaria3b.png
ka2_komarua1.png
ka2_komarua1b.png
ka2_majimea1.png
ka2_majimea1b.png
ka2_nayamua1b.png
ka2_odorokia1.png
ka2_odorokia1b.png
ka2_odorokia2.png
ka2_odorokia2b.png

(I'm guessing those are special sprites with changed eye color, might not be used except for a few scenes)

EDIT: After further reading the scene, it occurs to me most of the above sprites aren't shown at all. They seem to be defined as numalias in the script but are newer actually called in game...

**Comment by drojf** [comment]
Only one image had to be replaced in the end - `ka2_ikaria3.png`. There is no directly corresponding ps3 image (the ps3 ikari3 is super super angry), so I did what I think the umitweak team did, and photoshopped it. I have included two versions, pick whichever one you think looks better

http://imgur.com/a/IynIk

[ka2_ikaria3_alt_ver.zip](https://github.com/07th-mod/umineko-question/files/523850/ka2_ikaria3_alt_ver.zip)

For reference, to resize the sprite, I scaled the original ps3 height from 1140 pixels to 878 pixels to match all the other sprites.

----

ka2_defa2b.png - never used
ka2_fumana1b.png - never used
ka2_fumana2b.png - never used
ka2_ikaria1.png - never used
ka2_ikaria1b.png - never used
ka2_ikaria2.png - never used
ka2_ikaria2b.png - never used
ka2_ikaria3.png - used at Line 118750: ld c,$KA2_ikariA3,80
ka2_ikaria3b.png - never used
ka2_komarua1.png - never used
ka2_komarua1b.png - never used
ka2_majimea1.png - never used
ka2_majimea1b.png - never used
ka2_nayamua1b.png - never used
ka2_odorokia1.png - never used
ka2_odorokia1b.png - never used
ka2_odorokia2.png - never used
ka2_odorokia2b.png - never used

The only "KA2" sprites ever used are:

	Line 118746: ld c,$KA2_nayamuA1,6
	Line 118751: ld c,$KA2_ikariA3,80
	Line 118772: ld l,$KA2_fumanA2,6
	Line 118779: ld l,$KA2_fumanA1,80
	Line 124423: ld5 c,$KA2_defA1,99,2000,"breakup.dll/llB"
	Line 124436: ld c,$KA2_DefA2,80
	Line 124485: ld l,$KA2_defA1,0
	Line 124501: ld l,$KA2_DefA1B,1
	Line 124549: ld r,$KA2_defA1B,24
